evap/evaluation/models.py

```diff
@@ -59,7 +61,7 @@ def __str__(self):
 
     @property
     def can_be_deleted_by_manager(self):
-        return self.evaluations.count() == 0 or (self.participations_are_archived and self.grade_documents_are_deleted and self.results_are_archived)
+        return not self.is_active and (self.evaluations.count() == 0 or (self.participations_are_archived and self.grade_documents_are_deleted and self.results_are_archived))
```

Comment:
I think we can leave this like this for now, but in general, at some point it becomes more readable to split such statements like this:
```
if self.is_active:
    return False

return self.evaluations.count() == 0 or (self.participations_are_archived and self.grade_documents_are_deleted and self.results_are_archived)
```